views/pod.slim

```diff
@@ -38,6 +38,23 @@ ruby:
     eos
   end
 
+  stats = Hash[@stats.to_h.map do |k, v|
```

Comment:
in a helper method in the controller to process those hashes for consumption by there view